package/dist/lbcamden-frontend-0.5.3.min.js

```diff
@@ -1 +1,429 @@
+import * as GOVUK from "/govuk-frontend.min.js";
```

Comment:
could this be a relative import? would be nice not to require that this file is hosted at the root of the domain